tests/e2e/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# syntax=docker/dockerfile:1
+
+ARG BASE_IMG_TAG=nonroot 
+
+## Build Image
+FROM golang:1.18-bullseye as build
+
+WORKDIR /osmosis
+COPY . /osmosis
+
+# From https://github.com/CosmWasm/wasmd/blob/master/Dockerfile
+# For more details see https://github.com/CosmWasm/wasmvm#builds-of-libwasmvm 
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v1.0.0-beta7/libwasmvm_muslc.a /lib/libwasmvm_muslc.a
```

Comment:
So initially I checked the height via a `docker exec` through bash. This required an ubuntu image rather than distroless (the difference shown here). I just added a bunch of util functions and one of these utilizes a dockertest to check the height instead, which in turn allows me to use distroless again. So I will revert this